packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6040,6 +6040,20 @@ export default class Meeting extends StatelessWebexPlugin {
         EVENT_TRIGGERS.MEETING_MEDIA_REMOTE_STARTED,
         data
       );
+      if (data.type === 'share') {
```

Comment:
nitpick: not sure if it matters, but ideally we should be sending "client.media.render.start" after "client.media.rx.start", right? so this block should be 11 lines down.

Reply:
CA organizes the events as they come in, but I will change it to match the desired flow of events for readability